### DISCUSSION TITLE
Pyboard D-Series with adapter

### DISCUSSION THREAD

**Discussion by Jurkylius** [micropython/micropython#12190] [STM32 / Pyboard]
Hello guys,
I have maybe stupid question but need know what doing wrong.
I buy pyboard D-series and when I plug it with USB to PC work good but when I connect pyboard to adapter IC from WBUS then
not working when I plug it on PC. How much power need this devices togehter?

**Comment by peterhinch** [comment]
The Wbus adaptors on the MicroPython site use very little power. The board should work fine on USB.
